_candidates_en/YukiNakayama_2.md

```diff
@@ -1,6 +1,6 @@
 ---
 name: "Yuki Nakamura"
-title: "Revisiting ”I switched my job thanks to ScalaMatsuri”"
+title: "Revisiting "I switched my job thanks to ScalaMatsuri”"
```

Comment:
Due to the header formatting, we have to use the curly looking double quotation mark inside the straight quotes, I think.